CMakeLists.txt

```diff
@@ -61,8 +61,7 @@ if("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
     -Wlogical-op
   )
   set(PROJ_C_WARN_FLAGS ${PROJ_common_WARN_FLAGS}
-    # -Werror=declaration-after-statement
-    -Wdeclaration-after-statement
+    -Werror=declaration-after-statement
```

Comment:
looking at that more closely, we should drop that commit. We are now at C99 which allows declarations after statements, which isn't a bad practice

Reply:
I was thinking to mention this. It was added in 2013 to support old MSVC compilers, before C99 was required for PROJ.